modin/pandas/test/internals/test_benchmark_mode.py

```diff
@@ -13,19 +13,47 @@
 
 # test BenchmarkMode == True
 
+import pytest
+
 from contextlib import nullcontext
 import modin.pandas as pd
-from modin.pandas.test.utils import test_data_values
+from modin.pandas.test.utils import test_data_values, df_equals
 from modin.config import BenchmarkMode, StorageFormat
 from modin.test.test_utils import warns_that_defaulting_to_pandas
 
 
+default_to_pandas_context = (
+    warns_that_defaulting_to_pandas()
+    if StorageFormat.get() == "Omnisci"
+    else nullcontext()
+)
```

Comment:
This could (potentially) be wrong, as `StorageFormat` could be changed and a test could be called multiple times.
To avoid copy-pasting and retain the re-evaluation of `StorageFormat` you can wrap this in a function.

Reply:
Done, wrapped this check into a function and placed it in [`modin/pandas/test/utils.py`](https://github.com/dchigarev/modin/blob/539551e20faf933d382dc11d432ae1638f27273a/modin/pandas/test/utils.py#L1410)